src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Altinn.Platform.Authorization.IntegrationTest.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Altinn.Authorization.ABAC\Altinn.Authorization.ABAC.csproj" />
```

Comment:
I'm thinking this should be included as a nuget rather than a project reference as this is what is done for the component that's under testing

Reply:
Removing the line as I see the nuget is included further up

Reply:
this is already done by @Kritsi but my branch was created before her fix, so merging master into branch would fix it